server/opendp_apps/dataverses/views.py

```diff
@@ -96,3 +105,59 @@ def view_as_dict(request, object_id):
         return JsonResponse(get_json_error('Object not found'), status=HTTPStatus.NOT_FOUND)
 
     return JsonResponse(get_json_success('Success', data=mparams.as_dict()))
+
+
+class DataverseUserView(APIView):
+
+    def post(self, request):
+        opendp_user_id, dataverse_handoff_id = request.POST['user_id'], request.POST['dataverse_id']
+        dataverse_handoff = get_object_or_404(DataverseHandoff, id=dataverse_handoff_id)
+        api_general_token = dataverse_handoff.apiGeneralToken
+        site_url = dataverse_handoff.siteUrl
+        dataverse_client = DataverseClient(site_url, api_general_token)
+        try:
+            dataverse_response = dataverse_client.get_user_info(user_api_token=api_general_token)
+        except InvalidSchema:
+            return JsonResponse({'error': f'Site {site_url} is not valid'}, status=400)
+
+        if dataverse_response.success is not True:
+            print(dataverse_response.__dict__)
+            return JsonResponse({'error': dataverse_response.message}, status=400)
+
+        try:
+            handler = DataverseUserHandler(opendp_user_id, site_url, api_general_token, dataverse_response.__dict__)
+            new_dv_user = handler.create_dataverse_user()
+            new_dv_user.save()
+        except DataverseResponseError as ex:
+            print(ex)
+            return JsonResponse({'error': ex}, status=400)
+
+        print(dataverse_response.__dict__)
+        return JsonResponse({'dv_user': new_dv_user.id}, status=201)
+
+    def put(self, request):
+        content = JSONParser().parse(request)
```

Comment:
Similar to the above re: keys as entries in the static_vals file and error handling for non-existent keys